packages/@uppy/companion/src/config/grant.js

```diff
@@ -8,12 +8,14 @@ module.exports = () => {
         'https://www.googleapis.com/auth/drive.readonly',
       ],
       callback: '/drive/callback',
+      custom_params: { access_type : 'offline' },
```

Comment:
what is this?

Reply:
<img width="807" alt="Screenshot 2023-06-08 at 21 43 55" src="https://github.com/transloadit/uppy/assets/402547/da792f11-570b-48cc-9fc9-ba57da205bb8">